openshift/templates/api/api-deploy.param

```diff
@@ -8,21 +8,24 @@ APP_NAME=identity-kit
 APP_GROUP=identity-kit-controller
 SUFFIX=-bc
 ROLE=controller
+AGENT_ROLE=agent
+PUBLIC_FRONTEND_ROLE=public-frontend
+ADMIN_FRONTEND_ROLE=admin-frontend
 IMAGE_NAMESPACE=a99fd4-tools
 NAMESPACE_NAME=a99fd4
 SERVICE_PORT=8080
 DB_SERVICE=identity-kit-db
 DB_PORT=27017
 DB_NAME=identity_kit_db
-SOURCE_IMAGE_NAME=issuer-api
+SOURCE_IMAGE_NAME=issuer-kit-api
 TAG_NAME=test
 AGENT_SERVICE_NAME=identity-kit-agent
 AGENT_ADMIN_PORT=8024
 SMTP_SERVER_ADDRESS=
 SMTP_SERVER_PORT=
 ADMIN_EMAIL=
 EMAIL_SUBJECT=Invite to Identity Kit
-PUBLIC_SITE_URL=https://identity-kit-a99fd4-test.pathfinder.gov.bc.ca
+PUBLIC_SITE_URL=https://identity-kit-a99fd4-test.apps.silver.devops.gov.bc.ca
```

Comment:
nit.: can we please have the URLs not have the namespace in them? So somethign like `https://identity-kit-test.apps.silver.devops.gov.bc.ca`?

Reply:
change completed